app/controllers/validation/provider_packages_query_parameters.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# rubocop:disable Naming/VariableName
+
+module Validation
+  class ProviderPackagesQueryParameters
+    include ActiveModel::Validations
+
+    attr_accessor :sortFilter, :selectedFilter,
+                  :contentTypeFilter
+
+    validates :sortFilter, inclusion: { in: %w[name relevance],
+                                        message: 'Invalid Query Parameter for sort' }, allow_nil: true
+    validates :selectedFilter, inclusion: { in: %w[true false ebsco],
+                                            message: 'Invalid Query Parameter for filter[selected]' }, allow_nil: true
+    validates :contentTypeFilter, inclusion: { in: %w[aggregatedfulltext abstractandindex ebook ejournal print onlinereference unknown],
```

Comment:
Looks great @Sobha -- just one small think - I think we need to check for all as well